assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -163,7 +163,10 @@ function MediaPane(props) {
         allowedImageMimeTypes.includes(mimeType) ||
         allowedVideoMimeTypes.includes(mimeType)
     )
-    .map((attachment) => getResourceFromAttachment(attachment));
+    .map((resource) => {
+      if (resource.local) return resource;
```

Comment:
I commented on this above. IMHO this is a critical issue.

Reply:
Ditto: this is still an issue here. The `getResourceFromAttachment` below clearly demonstrates it.